components/bill/BillDetails.tsx

```diff
@@ -34,7 +34,8 @@ const StyledImage = styled(Image)`
 export const BillDetails = ({ bill }: BillProps) => {
   const { t } = useTranslation("common")
 
-  const isPendingUpgrade = useAuth().claims?.role === "pendingUpgrade"
+  const { user } = useAuth()  // Get the logged-in user (adjust according to your auth logic)
```

Comment:
This doesn't work because `user` isn't the root of the state returned by `useAuth()` (see https://github.com/codeforboston/maple/blob/main/components/auth/redux.ts#L17 for the structure of the auth state). The auth state has both `user` and `claims` as top-level keys, so trying to get `claims` from `user` instead in the `isPendingUpdate` check breaks the build.

This could be adjusted to work correctly by destructuring both `user` and `claims` from `useAuth()` (e.g. `
const { user, claims } = useAuth()`